### ISSUE TITLE
[suggestion] Auto-Refuse worn out clothing

### ISSUE THREAD

**Issue by Totherphoenix** [DFHack/dfhack#3358]
A function that checks for unworn clothing, armor, etc. with x, X, XX, etc. tag and marks them for dumping periodically throughout the fort, preventing the never-ending build-up of worn out clothing over time (primarily caused by the autoclothing and tailor features).

**Comment by myk002** [comment]
Note that tailor *already* marks work clothing for dumping when it wants to poke a dwarf to go get a replacement, so this would solve the cases that are *not* already handled by `tailor`.

I believe this use case will be handled by the new `stocks` interface being worked on in #3150. You'll be able to filter by wear and mark all for trade/dump

**Comment by glolsh** [comment]
but if I use autoclothing - how do I solve it?

**Comment by ab9rf** [comment]
isn't this just `cleanowned x` or `cleanowned X`?

**Comment by glolsh** [comment]
Yeah, but I need to manually enter it here and there. Is there a way like scheduled command execution? Or my best bet to add such commands to hotkey (keybind) and execute it from time to time?

**Comment by ab9rf** [comment]
> Yeah, but I need to manually enter it here and there. Is there a way like scheduled command execution? Or my best bet to add such commands to hotkey (keybind) and execute it from time to time?

I believe there is already a standard option in the control panel to run `cleanowned` once a month.

**Comment by glolsh** [comment]
Thank you!
